internal/devtools.py

```diff
@@ -241,6 +243,10 @@ def start_recording(self):
         self.send_command('Debugger.enable', {})
         self.send_command('ServiceWorker.enable', {})
         self.enable_target()
+        performance_metrics_script = os.path.join(self.script_dir, 'performance_metrics.js')
```

Comment:
Do we want to hard code this script into the agent, or should we use the same approach as custom metrics where it's passed from the server as a base64 encoded string?

Guess the question is how much work is it to deploy the agents?